hack/tests/e2e-ansible.sh

```diff
@@ -122,6 +122,8 @@ test_operator() {
         kubectl logs deployment/memcached-operator -c ansible
         exit 1
     fi
+    # If there are errors, that is ok. We just need to handle this correctly,
```

Comment:
Why this comment was added here? Should not the above validation be made?

Reply:
Removed.